### ISSUE TITLE
suggestion container is not disappearing on fast input

### ISSUE THREAD

**Issue by LexRiver** [Pixabay/jQuery-autoComplete#33]
just type completely some word from autocomplete list and press [tab] key fast. The suggestion container is not disappearing at all.

i've fixed it like this:
```javascript
            that.on('blur', function(){
                //console.log('blur');
                //that.updateSC();
                setTimeout(function(){
                    //console.log('hide me!');
                    that.sc.hide();
                }, 500);

            });
```
but I think author can make it much better.

**Comment by SimonSteinberger** [comment]
:+ I think that's actually a really good solution. Although such a timeout may look a bit weird, it's often the best way to handle such situations - at least in my experience! But it's probably necessary to call `that.updateSC();`at some point to make it all work properly.

I'll check that out very soon. Just on a tight schedule these days :-/

**Comment by SimonSteinberger** [comment]
Fixed in version 1.0.7, Thanks, LexRiver!
